Tasks/replace.js

```diff
@@ -1,22 +1,23 @@
 // Replace substring with newstr
+"use strict";
 
-Replace = (str, substr, newstr) => {
-  if (substr === '') {
+const replaceSubstring = (str, substr, newstr) => {
+  if (substr === "") {
+    return str;
+  }
+
+  const index = str.indexOf(substr);
+
+  if (index === -1) {
     return str;
   } else {
```

Comment:
```suggestion
  if (index === -1) return str;
```
We do not need `else` here because positive branch of if statement contains `return`